build/tools/certgen.sh

```diff
@@ -67,8 +67,9 @@ stream() {
     readonly STREAM_KEY_FILE=${certPath}/stream.key
     readonly STREAM_CSR_FILE=${certPath}/stream.csr
     readonly STREAM_CRT_FILE=${certPath}/stream.crt
-    readonly K8SCA_FILE=/etc/kubernetes/pki/ca.crt
-    readonly K8SCA_KEY_FILE=/etc/kubernetes/pki/ca.key
+
+    readonly k8sCaFile=${K8SCA_FILE:-/etc/kubernetes/pki/ca.crt}
+    readonly k8sCaKeyFile=${K8SCA_KEY_FILE:-/etc/kubernetes/pki/ca.key}
```

Comment:
why need to fix this? IMHO, K8SCA_FILE is never set. :)

Reply:
Consider the path of k8sCA maybe different, so optimize the code. If the file is never set, maybe the patch is not necessary.

Reply:
Maintain uniform variable naming rules?

Reply:
Modify k8sCaFile to K8SCA_FILE, k8sCaKeyFile to K8SCA_KEY_FILE? Like this?

Reply:
yeah, You can also change it all into humps.

Reply:
OK, I will fix this.